electrum/gui/qt/wizard/wizard.py

```diff
@@ -46,8 +46,10 @@ def __init__(self, config: 'SimpleConfig', app: 'QElectrumApplication', *, start
 
         self.back_button = QPushButton(_("Back"), self)
         self.back_button.clicked.connect(self.on_back_button_clicked)
+        self.back_button.setEnabled(False)
         self.next_button = QPushButton(_("Next"), self)
         self.next_button.clicked.connect(self.on_next_button_clicked)
+        self.back_button.setEnabled(False)
```

Comment:
copy-paste blindness! even as a reader I almost missed it

Reply:
good catch, thx